stock_putaway_product/views/product.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<openerp>
+    <data>
+
+        <!-- Product Form view -->
+        <!--
+            inheriting base structure of product.template,
+            common with product.product
+         -->
+        <record id="view_template_property_form" model="ir.ui.view">
+            <field name="name">nl.product.normal.form</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="stock.view_template_property_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//group[@name='store']/field[@name='loc_rack']"
+                       position="attributes">
+                    <attribute name="invisible">1</attribute>
+                </xpath>
+                <xpath expr="//group[@name='store']/field[@name='loc_row']"
+                       position="attributes">
+                    <attribute name="invisible">1</attribute>
+                </xpath>
+                <xpath expr="//group[@name='store']/field[@name='loc_case']"
+                       position="attributes">
+                    <attribute name="invisible">1</attribute>
+                </xpath>
+                <xpath expr="//group[@name='store']" position="inside">
+                    <field name="product_putaway_ids"
+                           context="{'default_product_template_id': active_id }"
```

Comment:
On a variant form view, active_id is  a product.product id.  So if I try to add a new storage location on a variant form, the "Product Variant" field choices contains variants of another template.